### PR TITLE
Code comments

### PR DESCRIPTION
**Pull request by dekatzenel**

### CODE REVIEW

File: parallel_tempering.py [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+            if prev_Es[i] < best_path_value:
+                best_path = Xs[i]
+                best_path_value = prev_Es[i]
+
+        # Decide which chains, if any, to exchange
+        if step % nbefore == 0:
+            for i in range(nsystems - 1, 0, -1):
+                # Acceptance probability
+                # 0 = ln(1)                
+                A = np.exp(0, ((delta_Es[i] - delta_Es[i-1])/Ts[i]) +
```
Comment (jonathanhfriedman) [inline parallel_tempering.py]: Should be
A = np.exp(min(0, ((delta_Es[i] - delta_Es[i-1])/Ts[i]) +
                              ((delta_Es[i-1] - delta_Es[i])/Ts[i-1])))

Comment (dekatzenel) [inline parallel_tempering.py]: That's where the extra parenthesis on the right came from!

**Comment by jonathanhfriedman** [comment]
Other than that one min(), everything seems to work!

**Comment by dekatzenel** [comment]
I'll merge it.

**Pull request merged**